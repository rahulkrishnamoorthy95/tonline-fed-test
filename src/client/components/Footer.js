import React from 'react';

const Footer = () => {
    return (
        <footer className="footer">
            <p>Designed By: Rahul </p>
        </footer>
    )
}

export default Footer;