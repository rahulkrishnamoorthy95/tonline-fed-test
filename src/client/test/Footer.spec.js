import React from 'react';
import Footer from '../components/Footer';

describe('Footer', () => {
	it('renders Designed By: Name', () => {
		const actual = shallow(<Footer />);
		expect(actual.get(0)).toMatchSnapshot();
	});
});