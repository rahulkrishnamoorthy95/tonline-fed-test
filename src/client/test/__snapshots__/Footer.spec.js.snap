// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer renders Designed By: Name 1`] = `
<footer
  className="footer"
>
  <p>
    Designed By: Rahul 
  </p>
</footer>
`;
